kuma/wiki/models.py

```diff
@@ -1743,11 +1743,8 @@ def previous(self):
         Returns the previous approved revision or None.
         """
         try:
-            return self.document.revisions.filter(
-                is_approved=True,
-                created__lt=self.created,
-            ).order_by('-created')[0]
-        except IndexError:
+            return self.get_previous_by_created(is_approved=True)
+        except self.DoesNotExist:
             return None
```

Comment:
I've tried that before in c715cac1bde384f3452633c97ede9725abf0a7a1 and it didn't work well, see https://bugzilla.mozilla.org/show_bug.cgi?id=1174820. Could you make sure that this doesn't happen again?